README.adoc

```diff
@@ -87,6 +87,29 @@ The `withGradle` wrapper should be used instead, since it also deals well with p
 
 image::find-build-scans.png[Find build scans,{thumbnail}]
 
+== Enriched build scans
```

Comment:
❓ Should we also mention that it will work in both cases if the user has configured GE manually or the auto-injection was used?